### PR TITLE
[FR] Update `Legal`, `Privacy`, add `Copyright`, `Music Licensing`, `Terms`

### PR DESCRIPTION
**Pull request by DoryanR**
Update of the translations of the following wiki pages:

- `Legal/Privacy`
- `Legal`

Translation of the following wiki pages:

- `Legal/Copyright`
- `Legal/Music_Licensing`
- `Legal/Terms`

### CODE REVIEW

File: wiki/Legal/Music_Licensing/fr.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+legal: true
+---
+
+# Licence musicale d'osu!
+
+100% du contenu d'osu! (généralement connu sous le nom de "[beatmaps](/wiki/Beatmap)" ou de niveaux de jeu) est téléchargé par les utilisateurs. Bien que nous essayions de guider les utilisateurs pour qu'ils obtiennent correctement l'autorisation, il y a souvent des cas de musique ou d'œuvres d'art téléchargées sans avoir obtenu correctement l'autorisation.
```
Comment (Satsukel) [inline wiki/Legal/Music_Licensing/fr.md:7]:
```suggestion
100% du contenu d'osu! (généralement connu sous le nom de "[beatmaps](/wiki/Beatmap)" ou de niveaux de jeu) est téléchargé par les utilisateurs. Bien que nous essayions de guider les utilisateurs pour qu'ils obtiennent correctement une autorisation, il y a souvent des cas de musique ou d'œuvres d'art téléchargées sans avoir obtenu correctement cette autorisation.
```

File: wiki/Legal/Music_Licensing/fr.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Les utilisateurs peuvent chercher et télécharger des niveaux à partir de la [liste des beatmaps](https://osu.ppy.sh/beatmapsets) en saisissant l'artiste, le titre ou toute autre information pertinente. La plupart des téléchargements de niveaux comprennent une chanson, des données de niveau créées par l'utilisateur, une image de fond et des effets sonores de jeu.
+
+Lorsqu'ils soumettent des niveaux, les créateurs sont invités à obtenir l'autorisation des détenteurs de droits d'auteur pour toutes les ressources, conformément à nos [directives d'utilisation du contenu](/wiki/Rules/Content_Usage_Guidelines).
+
+## Conditions de licence pour les artistes associés
+
+La licence musicale d'osu! peut être téléchargée gratuitement depuis la page [Artistes Associés](https://osu.ppy.sh/beatmaps/artists) dans des fichiers `.osz`, qui sont des modèles de beatmap. Un aperçu d'une minute généré automatiquement de chaque morceau est disponible en streaming sur la page de l'artiste. Les chansons étant utilisées comme un catalogue permettant aux membres de la communauté de créer des beatmaps, il se peut que certaines d'entre elles ne soient pas immédiatement utilisées comme contenu dans le jeu.
```
Comment (Satsukel) [inline wiki/Legal/Music_Licensing/fr.md:25]:
```suggestion
La licence musicale d'osu! peut être téléchargée gratuitement depuis la page [Artistes Associés](https://osu.ppy.sh/beatmaps/artists) dans des fichiers `.osz`, qui sont des modèles de beatmap. Un aperçu d'une minute généré automatiquement de chaque morceau est disponible en streaming sur la page de l'artiste. Les chansons étant utilisées comme un catalogue permettant aux membres de la communauté de créer des beatmaps, il se peut que certaines d'entre elles ne soient pas encore utilisées comme contenu dans le jeu.
```

**Review by Satsukel — CHANGES_REQUESTED** [review]

**Comment by DoryanR** [comment]
@Satsukel As soon as you are available I let you do a reckeck

**Review by Satsukel — CHANGES_REQUESTED** [review]

**Review by Satsukel — APPROVED** [review]

**Comment by DoryanR** [comment]
Can be merge 👍

**Review by TPGPL — APPROVED** [review]

**Pull request merged**